action.yml

```diff
@@ -53,6 +59,14 @@ runs:
     - name: Verify input
       run: echo 'Publishing SDK:' && echo ${{ inputs.sdk }}
       shell: bash
+    - name: Assert pre-release
+      if: assertPrerelease
+      run: |
+        if [[ "${{ inputs.version }}" =~ "^v[0-9]+\.[0-9]+\.[0-9]+$" ]] then
+        	echo "It looks like you have attempted to publish a non-alpha release by accident."
```

Comment:
I'd love there to be a link to this tool's README but am honestly not sure how to do it succinctly.